<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RankList</class>
 <widget class="QWidget" name="RankList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>300</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>RankList</string>
  </property>
  <widget class="QPushButton" name="pushButton8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>299</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align : left;background-color: rgb(255, 255, 255);
font: 63 11pt &quot;Adobe 繁黑體 Std B&quot;;
border-color: rgb(245, 245, 247);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>299</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 247);
font: 63 11pt &quot;Adobe 繁黑體 Std B&quot;;
border-color: rgb(245, 245, 247);
text-align : left;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton4">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>90</y>
     <width>299</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align : left;background-color: rgb(255, 255, 255);
font: 63 11pt &quot;Adobe 繁黑體 Std B&quot;;
border-color: rgb(245, 245, 247);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton5">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>120</y>
     <width>299</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 247);
font: 63 11pt &quot;Adobe 繁黑體 Std B&quot;;
border-color: rgb(245, 245, 247);
text-align : left;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton1">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>299</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 247);
font: 63 11pt ;
border-color: rgb(245, 245, 247);
text-align : left;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton6">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>150</y>
     <width>299</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align : left;background-color: rgb(255, 255, 255);
font: 63 11pt &quot;Adobe 繁黑體 Std B&quot;;
border-color: rgb(245, 245, 247);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>299</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 247);
font: 63 11pt &quot;Adobe 繁黑體 Std B&quot;;
border-color: rgb(245, 245, 247);
text-align : left;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>299</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 247);
font: 63 11pt &quot;Adobe 繁黑體 Std B&quot;;
border-color: rgb(245, 245, 247);
text-align : left;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>299</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align : left;background-color: rgb(255, 255, 255);
font: 63 11pt &quot;Adobe 繁黑體 Std B&quot;;
border-color: rgb(245, 245, 247);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>299</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align : left;background-color: rgb(255, 255, 255);
font: 63 11pt &quot;Adobe 繁黑體 Std B&quot;;
border-color: rgb(245, 245, 247);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton1</sender>
   <signal>clicked()</signal>
   <receiver>RankList</receiver>
   <slot>click1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton2</sender>
   <signal>clicked()</signal>
   <receiver>RankList</receiver>
   <slot>click2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton3</sender>
   <signal>clicked()</signal>
   <receiver>RankList</receiver>
   <slot>click3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton4</sender>
   <signal>clicked()</signal>
   <receiver>RankList</receiver>
   <slot>click4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton5</sender>
   <signal>clicked()</signal>
   <receiver>RankList</receiver>
   <slot>click5()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton6</sender>
   <signal>clicked()</signal>
   <receiver>RankList</receiver>
   <slot>click6()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>-12</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton7</sender>
   <signal>clicked()</signal>
   <receiver>RankList</receiver>
   <slot>click7()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton8</sender>
   <signal>clicked()</signal>
   <receiver>RankList</receiver>
   <slot>click8()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton9</sender>
   <signal>clicked()</signal>
   <receiver>RankList</receiver>
   <slot>click9()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>-5</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton10</sender>
   <signal>clicked()</signal>
   <receiver>RankList</receiver>
   <slot>click10()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>5</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>-6</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>click1()</slot>
  <slot>click2()</slot>
  <slot>click3()</slot>
  <slot>click4()</slot>
  <slot>click5()</slot>
  <slot>click6()</slot>
  <slot>click7()</slot>
  <slot>click8()</slot>
  <slot>click9()</slot>
  <slot>click10()</slot>
 </slots>
</ui>
